rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -513,6 +517,13 @@ impl CalculatorBase for LodeSphericalExpansion {
                             }
                         }
 
+                        // Add k = 0 contributions for (m, l) = (0, 0)
+                        let k0_contrib = &self.compute_k0_contributions();
+                        for (_property_i, [n]) in values.properties.iter_fixed_size().enumerate() {
+                            let n = n.usize();
+                            array[[sample_i, 0, n]] += global_factor * k0_contrib[[n]];
```

Comment:
Accessing the array should use `_property_i`, since some `n` might not have been requested by the user.

```suggestion
                            array[[sample_i, 0, _property_i]] += global_factor * k0_contrib[[n]];
```